### PR TITLE
23034 add the config

### PR DESCRIPTION
**Pull request by kzdev420**
*Issue #:* /bcgov/entity#23034

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by chenhongjing — COMMENTED** [review]
Does emailer need it? and the `entity-service-account` should be added to some vaults.json too.

File: queue_services/entity-emailer/src/entity_emailer/config.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ class _Config():  # pylint: disable=too-few-public-methods
     PAY_API_URL = os.getenv('PAY_API_URL', None)
     AUTH_URL = os.getenv('AUTH_URL', None)
     ACCOUNT_SVC_AUTH_URL = os.getenv('ACCOUNT_SVC_AUTH_URL', None)
+    AUTH_SVC_URL = os.getenv('AUTH_URL', 'https://')
```
Comment (vysakh-menon-aot) [inline queue_services/entity-emailer/src/entity_emailer/config.py:77]: we have it in line 75

Comment (kzdev420) [reply queue_services/entity-emailer/src/entity_emailer/config.py:77]: Yes, we have it. But, the emailer import the legal-api.service that use as AUTH_SVC_URL not AUTH_URL
https://github.com/bcgov/lear/blob/881ad4d19940bd0f7e119d4fe9e19a55d08a2582/legal-api/src/legal_api/services/bootstrap.py#L282

**Comment by kzdev420** [comment]
> Does emailer need it? and the `entity-service-account` should be added to some vaults.json too.

Currently, the emailer doesn't use it yet. but I thought it can be used in the future.

**Comment by chenhongjing** [comment]
> > Does emailer need it? and the `entity-service-account` should be added to some vaults.json too.
> 
> Currently, the emailer doesn't use it yet. but I thought it might be used in the future.

You should remove it since it's duplicate with previous line.

File: queue_services/entity-emailer/src/entity_emailer/config.py (hunk 1 above)
Comment (kzdev420) [reply queue_services/entity-emailer/src/entity_emailer/config.py]: removed

**Comment by kzdev420** [comment]
> > > Does emailer need it? and the `entity-service-account` should be added to some vaults.json too.
> > 
> > 
> > Currently, the emailer doesn't use it yet. but I thought it might be used in the future.
> 
> You should remove it since it's duplicate with previous line.

Removed

**Review by chenhongjing — APPROVED** [review]
Make sure you have added value to secrets & cronjob yaml files in OCP before it's merged.

**Pull request merged**